### DISCUSSION TITLE
PowerShell 질문입니다.

### DISCUSSION THREAD

**Discussion by kth9107** [tedilabs/fastcampus-devops#9] [Q&A] [answered]
파워쉘의 명령어가 
nvim: 'nvim' 용어가 cmdlet, 함수, 스크립트 파일 또는 실행할 수 있는 프로그램 이름으로 인식되지 않습니다. 이름이 정확
한지 확인하고 경로가 포함된 경우 경로가 올바른지 검증한 다음 다시 시도하십시오.
위치 줄:1 문자:1
+ nvim
+ ~~~~~
    + CategoryInfo          : ObjectNotFound: (whomi:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

과 같이 오류가 발생해서 
vim 을 다운로드 받아 vim ~/.aws/config 명령어를 실행해서 액세스 키를 입력했는데
쓸수 없다고 합니다. 

"~\.aws\config" E212: 쓸 파일을 열 수 없습니다
계속하려면 엔터 혹은 명령을 입력하십시오

해결책이 있을까요?

**Comment by posquit0** [comment, marked as answer]
안녕하세요, 우선 PowerShell을 이용하시면 안되고 WSL 터미널을 이용하셔야 합니다.
윈도우에 Ubuntu WSL을 설치하셨다면 Ubuntu 터미널에 접속하시고 우분투 터미널에서 `vim ~/.aws/config` 명령어를 통해 파일을 작성하시기 바랍니다.
혹시나 `~/.aws/` 디렉토리가 존재하지 않는다면, 우선 `mkdir ~/.aws` 명령어로 `~/.aws/` 디렉토리를 생성해주셔야 합니다.

**Reply by LSW4701** [reply]
도움됬습니다.
